src/aind_slims_api/operations/__init__.py

```diff
@@ -1,5 +1,11 @@
 """Init operations dir"""
 
 from .ecephys_session import EcephysSession, fetch_ecephys_sessions
+from .histology_procedures import SPIMHistologyExpBlock, fetch_histology_procedures
```

Comment:
Using absolute imports intra-package are recommended by pep.
https://github.com/python/peps/blob/main/peps/pep-0008.rst

Reply:
omg thanks for catching that!